portfolio/src/main/webapp/my-travels/travels.js

```diff
@@ -0,0 +1,29 @@
+const placesCoords = {
+                    "berlin" : {lat: 52.521999, lng: 13.413016},
+                    "default" : {lat: 52.048947, lng: 20.442758},
+                    "disneyland" : {lat: 48.869486, lng: 2.780874},
+                    "krakow" : {lat: 50.061683, lng: 19.937111},
+                    "kolobrzeg" : {lat: 54.186629, lng: 15.570729},
+                    "matterhorn" : {lat: 45.938319, lng: 7.731824},
+                    "zurich" : {lat: 47.349534, lng: 8.491872}
+                   }
+
+function setLocation(location) {
+  let coords = placesCoords[location];
```

Comment:
nit: all the other `let` can be `const`, too :)

Reply:
Done